ls1intum/Artemis
# adapt the new text assessment version to the SGI feature

### Checklist

### Motivation and Context


We would like to use the SGI feature for the new text assessment version as well.
### Description

Allow the new assessment layout to accept drop of the SGI element.
### Steps for Testing


1. Navigate to tutor course dashboard
2. Go to a text exercise that has a submission to assess
3. Change the url from  `/submissions/` to `/submissions-new/` 
4. Drag and drop an SGI to the feedback field of a text block
make sure the points and feedback test are adjusted as well as the total score
### Screenshots


![image](https://user-images.githubusercontent.com/50635005/80374743-74d86780-8897-11ea-8f48-c6ce13875371.png)
![image](https://user-images.githubusercontent.com/50635005/80374767-7efa6600-8897-11ea-9e97-92acaf99791d.png)
